player/sndmix.c

```diff
@@ -265,12 +266,22 @@ static inline void rn_process_envelope(song_voice_t *chan, int *nvol)
 		envvol = CLAMP(envvol, 0, 256);
 		vol = (vol * envvol) >> 8;
 	}
+	
+	*nvol = vol;
+}
 
-	// Panning Envelope
-	if ((chan->flags & CHN_PANENV) && (penv->pan_env.nodes)) {
-		int envpos = chan->pan_env_position;
+
+static inline void rn_process_pan_env(song_voice_t* chan, int *nvol) {
```

Comment:
`nvol` parameter can be removed - volume is never adjusted here.

Reply:
Thanks, removed